### PR TITLE
Addressed PR comments

### PR DESCRIPTION
**Pull request by pankajtandon**
Addressed PR comments from @snicoll

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ The use cases that can currently be addressed are below. See [Tests](stripe-serv
 ```
   mvn clean install
   
-  # Skip gpg signing 
+  //Skip gpg signing
```
Comment (snicoll) [inline readme.md:214]: That looks like an unrelated change.

**Review by snicoll — COMMENTED** [review]
LGTM (added a note). Thanks

File: readme.md (hunk 1 above)
Comment (pankajtandon) [reply readme.md:214]: it is. I snuck it in because my markdown plugin exposed it as an error.
